apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -39,6 +39,28 @@ const createTokenDisplayName = (tokenKey, tokenType) => {
         : capitalisedNameSegments.join(' ');
 };
 
+/**
+ * Splits a font/typography token into a parsable css value
+ * @param {object} token - the token value i.e. {"size": "48|56","weight": "ExtraBold"}
```

Comment:
super minor - wondering if it's worth adding a text-decoration token value in @param too since it's needed in this function 👍

Reply:
I think with JSDoc you can even give examples like `token.size`, `token.weight`, hopefully the syntax would allow for `token['text-decoration']` too

Reply:
Example from the docs:
```
/**
 * Assign the project to an employee.
 * @param {Object} employee - The employee who is responsible for the project.
 * @param {string} employee.name - The name of the employee.
 * @param {string} employee.department - The employee's department.
 */
Project.prototype.assign = function({ name, department }) {
    // ...
};
```